apps/mobile/src/features/Notifications/Notifications.container.tsx

```diff
@@ -1,10 +1,47 @@
+import React, { useCallback } from 'react'
+import { Switch } from 'react-native'
 import { View, Text } from 'tamagui'
-import React from 'react'
+
+import { useAppSelector, useAppDispatch } from '@/src/store/hooks'
+import { SafeListItem } from '@/src/components/SafeListItem'
+import {
+  selectAppNotificationStatus,
+  selectDeviceNotificationStatus,
+  toggleAppNotifications,
+} from '@/src/store/notificationsSlice'
 
 export const NotificationsContainer = () => {
+  const dispatch = useAppDispatch()
+  const isAppNotificationEnabled = useAppSelector(selectAppNotificationStatus)
+  const isDeviceNotificationStatus = useAppSelector(selectDeviceNotificationStatus)
+
+  const handleToggleAppNotifications = useCallback(() => {
+    if (!isDeviceNotificationStatus) {
+      // TODO: call enable device notifications method PR: 4837 instead
+      return
+    }
+    dispatch(toggleAppNotifications(!isAppNotificationEnabled))
+  }, [isAppNotificationEnabled, isDeviceNotificationStatus])
+
   return (
-    <View paddingHorizontal={'$3'} style={{ flex: 1 }}>
-      <Text>Notifications</Text>
+    <View paddingHorizontal="$4" marginTop="$2" style={{ flex: 1 }}>
+      <Text fontSize="$8" fontWeight={600} marginBottom="$2">
+        Notifications
+      </Text>
```

Comment:
I've created a `SectionTitle` component... Might be sending it on next PRs so you can use it